### ISSUE TITLE
RuGPT3XL Colab Notebook Doesn't Work

### ISSUE THREAD

**Issue by Marcus-Arcadius** [amrzv/awesome-colab-notebooks#3]
Keeps running into errors with installing deep speed, sparse attention ops and logits???

**Comment by amrzv** [comment]
Recently I've made pull request with fix for this notebook, the problem was in `cuda` version (11 needed).
You can support this pull request [here](https://github.com/sberbank-ai/ru-gpts/pull/65).
Also, you can run it directry from my branch: [![Try Model Generation In Colab!](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/amrzv/ru-gpts/blob/fix-generation-notebook/examples/ruGPT3XL_generation.ipynb)

**Comment by Marcus-Arcadius** [comment]
Thanks mate, this ended up working me, your an amazing legend and keep up the fantastic work 😃

**Comment by Marcus-Arcadius** [comment]
@amrzv I am getting this error now when opening the saved notebook?
![cudaerror](https://user-images.githubusercontent.com/78793842/124387259-26d3b500-dd21-11eb-9283-2541c9be4af4.png)

**Comment by amrzv** [comment]
> @amrzv I am getting this error now when opening the saved notebook?
> ![cudaerror](https://user-images.githubusercontent.com/78793842/124387259-26d3b500-dd21-11eb-9283-2541c9be4af4.png)

I've just ran notebooks and there were no errors
![image](https://user-images.githubusercontent.com/36787333/124390639-376e3600-dcf5-11eb-9be6-6401805dfd5d.png)

Try "Runtime -> Reset all runtimes" and running again.

**Comment by Marcus-Arcadius** [comment]
Thanks, it ended up working!
